docker-compose.yml

```diff
@@ -24,7 +24,6 @@ services:
                ./manage.py runserver 0.0.0.0:${API_PORT}'
     volumes:
       - ./api:/home/user/code
-      - '${HOME}/.aws:/home/user/.aws'
```

Comment:
This is the origin of the permissions bug described in Issue https://github.com/AlexsLemonade/scpca-portal/issues/843